frontend/src/types/question.d.ts

```diff
@@ -1,9 +1,16 @@
+export type Example = {
+  input: string;
+  output: string;
+};
+
 type Question = {
-  id: string;
+  _id?: string;
   title: string;
   complexity: string;
-  topics: Set<string>;
+  topics: string[];
```

Comment:
Oh, why don't you just declare it as `Topic[]`?

Reply:
hmm... don't see the need as topics are just an arr of string.

Reply:
Oh, but I thought inside this topics array, we only accept `Topic` (which is an enum) value?

Reply:
Oh right, but anyway the frontend will be fetching topics from the backend ya? so I guess I won't have the enum in frontend by then.

Reply:
Yes, I think we can use the assumption that the data retrieved from the database should be correct. So we can assume backend will pass valid topics to frontend, hopefully. Then by this assumption, there is no need to put `Topic[]` bah. Let's resolve this conversation :)